app/src/main/java/com/asfoundation/wallet/billing/partners/OemIdExtractorService.kt

```diff
@@ -1,20 +1,43 @@
 package com.asfoundation.wallet.billing.partners
 
 import android.content.Context
+import com.aptoide.injectorextractor.Extractor
 import io.reactivex.Single
+import java.io.File
 import java.util.*
 import java.util.zip.ZipFile
 
 
-class OemIdExtractorService(private val context: Context) {
+class OemIdExtractorService(context: Context) {
+
+  val extractorV1: OemIdExtractorV1 = OemIdExtractorV1(context)
+  val extractorV2: OemIdExtractorV2 = OemIdExtractorV2(context)
```

Comment:
These can be private.
To discuss: Should this be passed as dependencies in the constructor